### ISSUE TITLE
promise bug: cannot handle thenable job

### ISSUE THREAD

**Issue by jiangzidong** [jerryscript-project/jerryscript#2060]
```  javascript
var p = Promise.resolve(1).then(function(x) {
    print(x) // will print 1
    return Promise.resolve(2)
  }).then(function(x) {
    print(x) // should print 2, but now this will not be executed
  })
```

I will fix soon.

**Comment by akosthekiss** [comment]
Fixed by #2061

**Comment by cooboyno111** [comment]
I download the latest jerryscript code. Execute correctly. Next, wait for iotjs to change.
